src/workerd/api/node/tests/diagnostics-channel-test.js

```diff
@@ -108,3 +108,15 @@ export const test_tracing = {
     await Promise.all(promises.map((p) => p.promise));
   },
 };
+
+export const serFailureTest = {
```

Comment:
Now that we can have tests for tailing workers in workerd maybe we can get a test for the exception thrown to the tail worker?